docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -446,8 +446,8 @@ There might be an error in the console of your browser or in Bash / Terminal / C
 If the app launches but you are encountering errors with the UI itself, for example if fonts are not being loaded or if the code editor is not displaying properly, you may try the following troubleshooting steps at least once:
 
 ```shell
-# Remove all installed node modules
-rm -rf node_modules ./**/node_modules
+# Remove all installed Node modules
+npm run clean
```

Comment:
this does not remove `node_modules` from the root folder.

Reply:
Yes, that's true. I was trying to keep things as simple as possible but might have went too far in that direction. I'll figure out a more accurate way to say what it does and update this PR.

Reply:
@ValeraS and @raisedadead, which do you think sounds better?
- Remove all necessary Node modules
- Remove all Node modules installed by npm run bootstrap
- Remove all Node modules installed in subdirectories

Or do you have another way to word this? Seems like there are a lot of ways to describe this command, but I don't want to cause confusion

Reply:
I have added a suggestion in above. Note, this is because later we can expand the command to be able to reset other things later like clearing a cache, getting rid of temporary artifacts etc.

Reply:
Sounds great! Thanks for the suggestions and explanation. I'll make those changes right away